acros.tex

```diff
@@ -134,6 +134,7 @@
 \newacronym{PWAR}{PWAR}{Pratt and Whitney Aircraft Reactor}
 \newacronym{PWR}{PWR}{Pressurized Water Reactor}
 \newacronym{PyNE}{PyNE}{Python toolkit for Nuclear Engineering}
+\newacronym{PyRe}{PyRe}{Pyroprocessing Repository}
```

Comment:
Is that actually what this stands for? For some reason I thought it stood for something like (Py)ro (Re)processing module. Repository is an overloaded term (I assume you mean git repository, but nuclear engineers will assume you mean nuclear repository). Maybe Pyre was better after all... no acroynym needed.